client_icons.txt

```diff
@@ -111,53 +97,53 @@ oneteamiphone process-one.net
 oneteam oneteam
 osiris osiris
 ovi-chat chat.ovi.com,chat.nokia.com,nokiachat,ovi#contacts
-pandion pandion
+palm palm
 palringo palringo
-poezio slixmpp.com,poezio
-psiplus psi+,psi-dev
+pandion pandion
+pidgin pidgin,gaim
+pigeon pigeon
+pixart pixart
+poezio poezio,poez.io
+profanity profanity
+psiplus psi+,psi-plus,psi-dev,psidev
 psi psi,psi-im
-pidgin pidgin,d0bfd0b8d0b4d0b6d0b8d0bd 
-pyicq-t pyicqt.googlecode.com,icq#transport
-qip qip.ru
-qippda pda.qip.ru,qip pda
-qipmobile mobileqip.ru &&qip 
-qipinfium qip#infium,qip#2010,qip#2012,2010.qip.ru
+pyicq-t pyicq,icq#transport
+qip qip
+qq qq
 qutim qutim
-radio-t apps.radio-t.com
-riddim code.matthewwild.co.uk#riddim
+qxmpp qxmpp
+radio-t radio-t
+riddim riddim
 sameplace xmpp4moz,hyperstruct.net
-sapo sapo,messenger.sapo.pt
-sawim sawim.ru,sawim#ne
+sapo sapo
+sat sat,salutatoi,salut-a-toi
+sawim sawim,sawim#ne
 siejc siemens#native#jabber#client
 sim sim
-sip-communicator sip-communicator,sip#communicator,jitsi
+sleekxmpp sleekxmpp
+slick slick
+slixmpp slixmpp
 smack-api igniterealtime.org#smack
-snapi-snup-bot snapi-bot.googlecode.comgithub.com 
 sonic-revolution sonicrevolution
 spark spark,igniterealtime.org
 spectrum spectrum,binarytransport
 swift swift
-talisman-bot jabber-net.ru#talisman-bot,j-tmb.ru
-talkonaut talkonaut,google.com#1.0.0.84
-talkgadget.google.com talkgadget.google.com
-talk.google.com talk.google.com,google.com#1.0.0.104
-google.com google.com
+talkonaut talkonaut
 tkabber tkabber
-telepathy.freedesktop.org telepathy
-tigase tigase.org
+telepathy telepathy,empathy
+mozilla mozilla,thunderbird,instantbird
+tigase tigase
 trillian trillian
-ultimate-bot ultimate-bot.googlecode.com
-utah-bot jabbrik.ru,jabrvista.net.ru
-utalk palringo.com
+uwpx uwpx
+vacuum vacuum
+vk4xmpp vk4xmpp,simpleapps.ru
+vkontakte vkontakte,vk.com,pyvk
 weather.com jabber weather.com#transport
-webclient chat.jabbercity.ru,web-am31.dyndns-ip.com
-weonlydo weonlydo
-wod-xmpp weonlydo.com
+wodxmpp weonlydo
 wtw wtw
-vacuum vacuum
-vkontakte vk.com,pyvk-t,vkontakte
 yabber yabber
+yandex yandex
```

Comment:
it doen't exist anymore

Reply:
-> http://jabberworld.info/Jabber_caps

Reply:
Ya.Online is dead for years. This entry really has to be removed

Reply:
Btw, there is YaPlus, and it looks like there is some kind of jabber there, but I didn't tested